tmt/steps/provision/testcloud.py

```diff
@@ -227,9 +228,10 @@ class ProvisionTestcloud(tmt.steps.provision.ProvisionPlugin):
     _keys = ["image", "user", "memory", "disk", "connection", "arch"]
```

Comment:
@janhavlin plesae check artemis.py, we now are doing this differently, via GuestSshData inheritance

Reply:
nm, this is StepData, sorry!